### PR TITLE
Add SCEP attribute type mappings

### PR DESCRIPTION
**Pull request by laurivosandi**
This is the very first step towards scepbuilder :)

### CODE REVIEW

**Comment by laurivosandi** [comment]
Hi, it's been almost two weeks. What's the status?

**Comment by wbond** [comment]
Status: busy family/personal life with a few other things having taken up the limited free time I have. I looked a little into the PR, and things looked ok from the implementation perspective, but I want to verify the ASN.1 specs at the source, which i haven't done yet.

File: asn1crypto/cms.py [hunk 1]
```diff
@@ -101,6 +102,13 @@ class CMSAttributeType(ObjectIdentifier):
         '1.2.840.113549.1.9.16.2.14': 'signature_time_stamp_token',
         # https://tools.ietf.org/html/rfc6211#page-5
         '1.2.840.113549.1.9.52': 'cms_algorithm_protection',
+        # http://www.cisco.com/c/en/us/support/docs/security-vpn/public-key-infrastructure-pki/116167-technote-scep-00.html
```
Comment (wbond) [inline asn1crypto/cms.py]: This should reference `https://tools.ietf.org/html/draft-gutmann-scep-06#section-3.2.1`

File: asn1crypto/cms.py [hunk 2]
```diff
@@ -101,6 +102,13 @@ class CMSAttributeType(ObjectIdentifier):
...
+        '2.16.840.1.113733.1.9.2': 'message_type',
+        '2.16.840.1.113733.1.9.3': 'pki_status',
+        '2.16.840.1.113733.1.9.4': 'fail_info',
+        '2.16.840.1.113733.1.9.5': 'sender_nonce',
+        '2.16.840.1.113733.1.9.6': 'recipient_nonce',
+        '2.16.840.1.113733.1.9.7': 'trans_id',
```
Comment (wbond) [inline asn1crypto/cms.py]: According to the latest draft, this should probably called `transaction_id`.

**Comment by wbond** [comment]
@laurivosandi Just checking in to see if you are still interested in this PR.

**Comment by laurivosandi** [comment]
Hi,

yes still interested, going to take another look soon

**Pull request closed without merging**